versioned_docs/version-2.0.x/reference/goals/awslambda.mdx

```diff
@@ -13,14 +13,14 @@ pants awslambda [args]
 
 Deprecated in favor of the `package` goal.
 
-Backend: <span className="color--primary">`pants.core`</span>
-
-Config section: <span className="color--primary">`[awslambda]`</span>
+Backend: <span className="color--primary"></span>
+Config section: <span className="color--primary">[awslambda]</span>
```

Comment:
I wonder if this spacing change will cause problems?

Reply:
Fixed in https://github.com/pantsbuild/pantsbuild.org/pull/53